src/src/DataTypes/SPI_options.h

```diff
@@ -0,0 +1,18 @@
+#ifndef DATASTRUCTS_SPI_OPTIONS_H
+#define DATASTRUCTS_SPI_OPTIONS_H
+
+#include <Arduino.h>
+
+enum class SPI_Options_e {
```

Comment:
The values of his enum are stored, so better add a comment to not change the order (or add the numerical values)